headers.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+const (
+	contentType = "application/json"
+	userAgent   = "Mozilla/5.0 (platform; rv:geckoversion) Gecko/geckotrail Firefox/firefoxversion"
+)
+
+var commonHeaders = map[string][]string{
+	"Accept":          {"*/*"},
+	"Accept-Encoding": {"gizp", "deflate", "br"},
+	"cache-control":   {"no-cache", "no-store", "must-revalidate", "post-check=0", "pre-check=0"},
+	"Connection":      {"keep-alive"},
+	"dnt":             {"1"},
```

Comment:
is this a "do-not-track" header added by your browser?

Reply:
I had no idea that `dnt` means do-not-track. My client adds it because it is imitating my Chrome browser.